.github/workflows/release.yml

```diff
@@ -24,6 +24,51 @@ jobs:
       - id: changelog-version
         run: echo "version=$(echo "${{ inputs.versionNumber }}" | cut -c 2-)" >> "$GITHUB_OUTPUT"
 
+  releasedVersion:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.remove-leading-v.outputs.version }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-ecosystem/action-get-latest-tag@v1
+        id: get-latest-tag
+        with:
+          semver_only: true
+
+      - name: Remove leading v from tag
+        id: remove-leading-v
+        run: |
+          VERSION='${{ steps.get-latest-tag.outputs.tag }}'
+          echo "VERSION=${VERSION:1}" >> $GITHUB_OUTPUT
+
+  buildBinary:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+      - name: Set up Go
+        uses: actions/setup-go@fac708d6674e30b6ba41289acaab6d4b75aa0753 # v4.0.1
+        with:
+          go-version-file: go.mod
+          cache: true
+
+      - name: Sync dependencies
+        shell: bash
+        run: |
+          go mod download
+          go mod tidy
+
+      - name: Build
+        run: |
+          go build -o terraform-provider-null
+
+      - name: Upload test artifacts
+        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        with:
+          name: terraform-provider-null
+          path: terraform-provider-null
+          retention-days: 1
```

Comment:
GitHub really makes it annoying to get just the name of the repository. `${{ github.event.repository.name }}` might work in this case. That said, since this binary is not needed elsewhere in the workflow, is there any reason to not include the build in the cdktf documentation job and skip the artifact handling?